### ISSUE TITLE
Document

### ISSUE THREAD

**Issue by immavalls** [grafana/k6-docs#1038]
Discussing with @na-- this forum https://community.k6.io/t/oneof-protobuf-type-not-working-grpc/5885, we think it would be good to document the behavior of `oneof` as he found in https://stackoverflow.com/questions/44069789/what-is-the-expected-json-serialization-of-oneof-protobuf-field.

Probably add a new section under https://k6.io/docs/javascript-api/k6-net-grpc/client/client-invoke/ and link to it from the request parameter explanation in https://k6.io/docs/javascript-api/k6-net-grpc/client/client-invoke/.

**Comment by jamie-macdonald** [comment]
Would be good to document the Any type here as well.  If this is an open source project do you want me to document it?  We could also do with some documentation for sharing a connection over all users and having a connection per virtual user.  I have eventually after a lot of playing got it to work.  Happy to share my findings.  At first I was using an array but this is not necessary as a variable in the init does the job.

**Comment by immavalls** [comment]
Thanks @jamie-macdonald! Feel free to create a PR with your findings.

**Comment by jamie-macdonald** [comment]
is the website held in code?

**Comment by immavalls** [comment]
Yes, it's in this same repo, you can read the contributors guide: https://github.com/grafana/k6-docs/blob/main/CONTRIBUTING/README.md
